packages/server/events-processing-platform/subscriptions/graph/service_line_item_event_handler_it_test.go

```diff
@@ -366,18 +384,22 @@ func TestServiceLineItemEventHandler_OnUpdatePriceIncrease_TimelineEvent(t *test
 	require.Equal(t, entity.DataSource(constants.SourceOpenline), action.Source)
 	require.Equal(t, constants.AppSourceEventProcessingPlatform, action.AppSource)
 	require.Equal(t, entity.ActionServiceLineItemPriceUpdated, action.Type)
-	require.Equal(t, "increased the price for SLI Price Increase from 150.00 / MONTHLY to 200.00 / MONTHLY", action.Content)
-	require.Equal(t, `{"price":200}`, action.Metadata)
+	require.Equal(t, "logged-in user retroactively increased the price for SLI Price Increase from 150.00 / monthly to 200.00 / monthly", action.Content)
```

Comment:
can we check with Antoine these messages? increased the price for SLI Price Increase doesn't sound too good i think